hack/update-integration-compatibility-versions-test.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
```

Comment:
`update-...` seems to imply that this mutates files. This only prints the gates and doesn't update the file right? Maybe something like `generate-...` would be more accurate? cc @BenTheElder